### PR TITLE
Created an audio player that plays my favourite music.

### PR DESCRIPTION
**Pull request by eriyezab**
The music files are in mp3 format and are located in a directory named audio. Apologies if spelling favourite like this bothers you as I am Canadian lol

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -26,3 +26,26 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+function audioPlayer() {
+  console.log('Running audio player.');
+  currentSong = 0;
+  player = document.getElementById('audio-player');
+  songs = document.querySelectorAll('#song-list li a');
+  console.log('Adding Event Listener to song links...');
```
Comment (shrutip90) [inline portfolio/src/main/webapp/script.js]: This doesn't seem very useful unless you expect the addEventListener code to fail or not run for some reason.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,26 @@ function addRandomFact() {
...
+  songs.forEach((song, index, songList) => {
+    song.addEventListener('click', (event) => {
+      console.log('Clicked song.');
+      event.preventDefault();
+      songList[currentSong].parentNode.className = '';
```
Comment (shrutip90) [inline portfolio/src/main/webapp/script.js:48]: Can you use index here instead?

Comment (shrutip90) [inline portfolio/src/main/webapp/script.js:48]: Just curious, why is this required?

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (eriyezab) [reply portfolio/src/main/webapp/script.js]: When I was making the function I was running into issues and so I had added it there in order to debug and forgot to take it out. I realized that the problem was that inside the index.html the script tag was at the top and so it was not working since the document hadn't loaded.

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (eriyezab) [reply portfolio/src/main/webapp/script.js:48]: This was so that when I change the song, the previous song no longer had a class name of current song and I was planning on styling it such that the current song was in a different colour than the others but I hadn't gotten to that yet. I should have made more comments clarifying this

Comment (shrutip90) [reply portfolio/src/main/webapp/script.js:48]: Okay, thanks. One suggestion would be to avoid adding changes that are unused currently - like you could add the 'current-song' class in the pull request where you are also adding the styling, so that it makes more sense :).

Comment (shrutip90) [reply portfolio/src/main/webapp/script.js:48]: Nvm, I guess that wouldn't work.

**Review by shrutip90 — APPROVED** [review]
Thanks for adding more comments!
